_posts/en/newsletters/2024-08-16-newsletter.md

```diff
@@ -0,0 +1,287 @@
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+<!-- FIXME:Gustavojfe -->
+
+- [Bitcoin Core #29519][] resets the `pindexLastCommonBlock` value after an
+  [assumeUTXO][topic assumeutxo] snapshot is loaded, so that a node prioritizes
+  downloading blocks after the most recent block in the snapshot. This fixes a
+  bug where a node would set `pindexLastCommonBlock` from existing peers before
+  loading the snapshot and start downloading blocks from that block, defeating
+  the purpose of assumeUTXO.
+
+- [Bitcoin Core #30598][] removes block height from the [assumeUTXO][topic
+  assumeutxo] snapshot file metadata as it is not a unique identifier in a
+  pre-sanitized untrusted file compared to the block hash that is kept. A node
+  can still obtain block height from many other internal sources.
+
+- [Bitcoin Core #28280][] optimizes initial block download (IBD) performance for
+  pruned nodes by not emptying the UTXO cache during pruning flushes. It does
+  this by listing "dirty" cache entries, entries that have changed since they
+  were last written to the database, separately. This allows a node to avoid
+  unnecessarily scanning the entire cache during prune flushes and instead focus
+  on the dirty entries.  The optimization results in up to 32% faster IBD for
+  pruned nodes with high `dbcache` settings, and about 9% improvement with
+  default settings. See Newsletter [#304][news304 cache].
+
+- [Bitcoin Core #28052][] adds XOR encoding to `blocksdir *.dat` files on
+  creation as a preventative mechanism against unintentional and accidental data
+  corruption by anti-virus software. This can be optionally disabled and doesn't
+  protect against intentional data corruption attacks. This was implemented for
+  the `chainstate` files in [Bitcoin Core #6650][] in September 2015.
+
+- [Core Lightning #7528][] adjusts the [fee rate estimation][topic fee
+  estimation] for sweeps of non-time-sensitive unilateral channel closures to an
+  absolute deadline of 2016 blocks (approximately 2 weeks). Previously, the fee
+  rate was set to target confirmation within 300 blocks, which caused
+  transactions to be stuck at the minimum relay fee limit, resulting in
+  indefinite delays.
+
+- [Core Lightning #7533][] updates the internal coin movement notifications and
+  the transaction bookkeeper to properly account for the spending of funding
+  outputs for [splicing][topic splicing] transactions. Previously, there was no
+  logging or tracking of this.
+
+- [Core Lightning #7517][] askrene
+
+- [LND #8955][] yyforyongyu/cr-8516-240729-sendcoins
+
+- [LND #8886][] bitromortac/buildroute-inbound-fees
+
+- [LND #8967][] add wire messages for quiescence
+
+- [LDK #3215][] tx-sync: Protect against Core's Merkle leaf node weakness
+
+- [BIPs #1658][] BIP94: Change timewarp delta to 7200 seconds
```

Comment:
Note https://github.com/bitcoin/bips/pull/1660

Reply:
if this is getting reverted, I guess we should remove this entry?

Reply:
Yeah, I think so